doc/src/manual/conversion-and-promotion.md

```diff
@@ -92,11 +95,11 @@ ERROR: MethodError: Cannot `convert` an object of type String to an object of ty
 [...]
 ```
 
-Some languages consider parsing strings as numbers or formatting numbers as strings to be conversions
-(many dynamic languages will even perform conversion for you automatically), however Julia does
-not: even though some strings can be parsed as numbers, most strings are not valid representations
-of numbers, and only a very limited subset of them are. Therefore in Julia the dedicated `parse`
-function must be used to perform this operation, making it more explicit.
+Some languages consider parsing strings as numbers or formatting numbers as strings to be
+conversions (many dynamic languages will even perform conversion for you automatically).
+Even though some strings can be parsed as numbers, most strings are not valid
```

Comment:
I think the distance between "some languages" and julia is too far to retain a consistent idea, though I do a gree the previous formulation was poor. Will include an improvement in #34244.